engine/src/flutter/lib/ui/painting.dart

```diff
@@ -163,19 +195,28 @@ class Color {
       g = (g & 0xff) / 255,
       b = (b & 0xff) / 255;
 
-  /// The alpha channel of this color.
+  /// The normalized alpha channel of this color.
   ///
-  /// A value of 0.0 means this color is fully transparent. A value of 1.0 means
-  /// this color is fully opaque.
+  /// A value of `0.0` means this color is fully transparent. A value of `1.0`
+  /// means this color is fully opaque.
   final double a;
 
-  /// The red channel of this color.
+  /// The normalized red channel of this color.
+  ///
+  /// A value of `0.0` represents no red in this color. A value of `1.0`
+  /// represents the maximum amount of red.
   final double r;
 
-  /// The green channel of this color.
+  /// The normalized green channel of this color.
+  ///
+  /// A value of `0.0` represents no red in this color. A value of `1.0`
+  /// represents the maximum amount of green.
   final double g;
 
   /// The blue channel of this color.
+  ///
+  /// A value of `0.0` represents no blue in this color. A value of `1.0`
+  /// represents the maximum amount of blue.
```

Comment:
I was very careful about using the word "normalized" or saying 1.0 is the maximum value since internally that isn't the case with ColorSpace.extendedSRGB.  It is normalized but 1.0 isn't the max value.  Can we remove that sentence please?